src/App.vue

```diff
@@ -54,7 +54,8 @@ export default {
           showLabels: this.paramsOptions?.showLabels ?? true,
           startingQuality: this.paramsOptions?.startingQuality,
           hideToast: this.paramsOptions?.hideToast,
-          mainLabel: this.paramsOptions?.mainLabel ?? 'Main'
+          mainLabel: this.paramsOptions?.mainLabel ?? 'Main',
+          enableDRM: this.paramsOptions.enableDRM ?? false
```

Comment:
@gregdolby I made this comment in the other PR as well, but if we want to keep casing consistent with the other params this should be `enableDrm`

Reply:
`enableDrm` is what I originally had I will change it back. The reason I switched was because connectionOptions (config for sdk) uses enableDRM but I will talk with Vincent to align this.